com.unity.cinemachine/Runtime/Core/SplineContainerExtensions.cs

```diff
@@ -138,5 +145,190 @@ static class SplineContainerExtensions
                 t += max;
             return t;
         }
+        
+        /// <summary>
+        /// Converts the distance from one unit to another.
+        /// </summary>
+        /// <param name="spline">The spline which length is used for conversion.</param>
+        /// <param name="distance">The distance to convert.</param>
+        /// <param name="oldUnits">The original units of the distance.</param>
+        /// <param name="newUnits">The units to convert the distance to.</param>
+        /// <returns> The converted distance.</returns>
+        [MethodImpl(methodImplOptions: AggressiveInlining)]
+        internal static float ConvertDistance(this Spline spline, float distance, PathIndexUnit oldUnits, PathIndexUnit newUnits)
```

Comment:
This re-implements functionality that is already available in SplineUtility.ConvertIndexUnit

Reply:
I overlooked this functionality entirely. Thanks!